brave-lists/brave-sugarcoat.txt

```diff
@@ -1,10 +1,2 @@
-||googletagservices.com/tag/js/gpt.js$script,important,domain=vimeo.com,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-83c6e054eb06b453d56c97b9ddd181d6.js
-||z.moatads.com/reutersheader194883552024/moatheader.js$script,important,domain=reuters.com,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-7754008b0da848856cddb20094976335.js
-||sourcepointcmp.bloomberg.com/ccpa.js$script,important,domain=bloomberg.com,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-1e156c8652627e444cc63b54d4de0ffb.js
-||quantcast.mgr.consensu.org/tcfv2/23/cmp2.js$script,important,domain=researchgate.net,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-1290d064f0a1f106f4c14d0dbd0aca60.js
-||assets.adobedtm.com/launch-ENb97d7f9d2d4b4720ac9782a711994995.min.js$script,important,domain=amazon.jobs,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-dd32890a88dc14e37c80777ca67c1e2e.js
-||assets.adobedtm.com/launch-EN3932511771fb4e5e9dd852ae89372b59.min.js$script,important,domain=fedex.com,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-7754fade75bed02e4097bf3d7fff178c.js
-||assets.adobedtm.com/extensions/EPbde2f7ca14e540399dcc1f8208860b7b/AppMeasurement_Module_AudienceManagement.min.js$script,important,domain=bmw.com.au,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-4c15a27239807d1629304fcae3cf6ce3.js
-||assets.adobedtm.com/extensions/EPbde2f7ca14e540399dcc1f8208860b7b/AppMeasurement_Module_AudienceManagement.min.js$script,important,domain=virginmobile.ca,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-d357b361eae50395667a4c91732192fd.js
-||quantcast.mgr.consensu.org/tcfv2/cmp2.js?referer=einthusan.tv$script,important,domain=einthusan.tv,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-968076ccecd06aaae1ffcb1e3d26399f.js
-||googletagservices.com/tag/js/gpt.js$script,important,domain=downdetector.com,redirect-url=https://pcdn.brave.com/sugarcoat/async-sugarcoat-a8badcefe5197e12ea29f0f5d0db1cc9.js
+||assets.adobedtm.com/extensions/EPbde2f7ca14e540399dcc1f8208860b7b/AppMeasurement_Module_AudienceManagement.min.js$script,important,domain=bmw.com.au,redirect=async-sugarcoat-4c15a27239807d1629304fcae3cf6ce3.js
```

Comment:
I believe the redirect filter option does not take in the file extension (.js), so the filter option would just be `redirect=async-sugarcoat-4c15a27239807d1629304fcae3cf6ce3`. However, I think these 2 rules are obsolete: https://github.com/brave/brave-browser/issues/18542#issuecomment-975628033

I have a local PR that adds a few new ones (10 rules), I can update that to be just 2 rules. I thought we were waiting for https://github.com/brave/brave-core/pull/10994 to be closed?

Reply:
> I believe the redirect filter option does not take in the file extension (.js)

For full compatibility with uBO, `redirect` accepts the version of the redirect resource with the `.js` extension or without. Although you're correct in that it's usually omitted.

> I thought we were waiting for brave/brave-core#10994 to be closed?

That PR is still being reviewed; we'd like to get some demo working sooner so as small of a subset as possible would be ideal.

> I have a local PR that adds a few new ones (10 rules), I can update that to be just 2 rules.

That'd be great!

Reply:
I updated https://github.com/brave/adblock-lists/pull/683/files, and we would also need https://github.com/brave/adblock-resources/pull/51